src/pages/plugins/index.js

```diff
@@ -104,28 +104,35 @@ function PluginComponent({ plugin }) {
   return (
     <div className="row padding-vert--md">
       <div className="col col--10 col--offset-1">
-        <div key={plugin.key} className={ plugin.deprecatedMessage ? `card ${classnames('card__deprecated', styles.pluginDeprecatedCard)}` : "card"}>
-          {!plugin.deprecatedMessage || (
+        {plugin.deprecatedMessage ? (
+          <div key={plugin.key} className={`card ${classnames('card__deprecated', styles.pluginDeprecatedCard)}`}>
+            <div className={classnames('card__header', styles.pluginCardHeader)}>
+              <h3 className={classnames('deprecated-banner', styles.deprecated)}>Deprecated</h3>
```

Comment:
This should just being using a badge, the same as in the non-deprecated plugin.

Reply:
This comment has solved 
`<span className="badge badge--primary">Deprecated</span>`